Cows-HacknSlash/Assets/Scripts/Inventory script/Inventory.cs

```diff
@@ -34,6 +52,7 @@ public void Additem(int id , int amount)
                 //spawn the object in that slot+
                 Instantiate(itemdisplayerprefab, slot.transform.localPosition, Quaternion.identity, slot.gameObject.transform);
 
+                //making sure the slot is displayed as something in it
```

Comment:
Please make all comments start with a space, then capital letter and end with a full stop/period (check all other comments too)